mbuild/tests/test_json_formats.py

```diff
@@ -17,6 +17,15 @@ def test_loop(self, ethane):
         assert ethane.n_bonds == ethane_copy.n_bonds
         assert len(ethane.children) == len(ethane_copy.children)
 
+    def test_box(self):
```

Comment:
Lets also have a test for the case where there is no box. After that, this LGTM!

Reply:
Test added